.percy.yml

```diff
@@ -1,5 +1,6 @@
 version: 2
 snapshot:
-  widths: [1280]
+  widths: [320, 602, 1024]
```

Comment:
Imo lets drop it to two sizes. We don't need 602 or 1024. 
We don't get much for now. It might be good for flex stuff but lets leave it for now.

Reply:
Hmm...I thought we had agreed on 3 sizes offline. Alright, I'm fine dropping one. I'll let @ianmcburnie decide which one to drop.

❗ It's important to set this to something we're fairly confident about keeping, at least for the near term. I believe every change we make to these will require a full run of snapshot baselines.

Reply:
If we have to drop one, let's drop 1024, as we don't add any additional breakpoint styles above 602. Layout wise things should follow the same rules at 1024 as 602.